lldb/source/Target/Statistics.cpp

```diff
@@ -236,6 +236,27 @@ void TargetStats::IncreaseSourceRealpathCompatibleCount(uint32_t count) {
 
 bool DebuggerStats::g_collecting_stats = false;
 
+void DebuggerStats::ResetStatistics(Debugger &debugger, Target *target) {
+  const uint64_t num_modules = target != nullptr
```

Comment:
Need to lock the `Module::GetAllocationModuleCollectionMutex()` when using the Module static calls for the global module list.